noxfile.py

```diff
@@ -259,19 +259,19 @@ def _install_requirements(session, transport, *extra_requirements):
             "Skipping Python Requirements because SKIP_REQUIREMENTS_INSTALL was found in the environ"
         )
         return
+
+    # setuptools 50.0.0 is broken
```

Comment:
You should be getting this change for "free" if you rebase against the current master branch.